nextflow_schema.json

```diff
@@ -58,10 +58,10 @@
                 "aligner": {
                     "type": "string",
                     "description": "Name of the tool to use for scRNA (pseudo-) alignment.",
-                    "default": "alevin",
-                    "help_text": "The workflow can handle three types of methods:\n\n- Kallisto/Bustools\n- Salmon + Alevin-fry + AlevinQC\n- STARsolo\n\nTo choose which one to use, please specify either `alevin`, `star` or `kallisto` as a parameter option for `--aligner`. By default, the pipeline runs the `alevin` option. Note that specifying another aligner option also requires choosing appropriate parameters (see below) for the selected option.",
+                    "default": "simpleaf",
+                    "help_text": "The workflow can handle three types of methods:\n\n- Kallisto/Bustools\n- SimpleAF \n- STARsolo\n\nTo choose which one to use, please specify either `simpleaf`, `star` or `kallisto` as a parameter option for `--aligner`. By default, the pipeline runs the `simpleaf` option. Note that specifying another aligner option also requires choosing appropriate parameters (see below) for the selected option.",
                     "fa_icon": "fas fa-align-center",
-                    "enum": ["kallisto", "star", "alevin", "cellranger", "cellrangerarc", "cellrangermulti"]
+                    "enum": ["kallisto", "star", "simpleaf", "cellranger", "cellrangerarc", "cellrangermulti"]
```

Comment:
I think to make the transition a bit smoother, I would allow`alevin` still as an option and mention it here, and in the pipeline code then print a warning and interpret it as `simpleleaf` so people can still use it.

Reply:
it was the default value, so there might have been quite some users, I assume

Reply:
Then let's keep support for `"alevin"` as a deprecated alias for `"simpleaf"`. @fmalmeida, could you add that to your open PR?

Reply:
Is it possible to add deprecated for a single "enum" option within a parameter entry?

Reply:
Should I just add in the `enum` and then, add a `log.info` telling people that is deprecated alias and will be removed in future version?

Reply:
I can do it already here in this PR for addressing the review comments: https://github.com/nf-core/scrnaseq/pull/443

Reply:
sounds good!

Reply:
Would be an option. Then would only need to merge #443 with it and have a new bump-version.

Reply:
yeah, let's just do that I guess.

Reply:
Going for major release:
https://github.com/nf-core/scrnaseq/pull/446